### PR TITLE
Add documentation for cakeissues.net

### PR DESCRIPTION
**Pull request by pascalberger**
Add documentation for cakeissues.net, moved from Cake.Issues.Website repository including full history.

Part of https://github.com/cake-contrib/Cake.Issues.Website/issues/123

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.RuleUrl`                  | Support for custom rules can be added through a custom [MarkdownlintAddRuleUrlResolver] |
+
+[Cake.Issues.Markdownlint addin]: https://www.nuget.org/packages/Cake.Issues.Markdownlint
+[Markdownlint]: https://github.com/DavidAnson/markdownlint
+[markdownlint-cli]: https://github.com/igorshubovych/markdownlint-cli
+[Cake.Markdownlint]: https://www.nuget.org/packages/Cake.Markdownlint/
+[MarkdownlintAddRuleUrlResolver]: ../../../api/Cake.Issues.Markdownlint/MarkdownlintIssuesAliases/2EE35F55
+[MarkdownlintLogFileFormat]: ../../../api/Cake.Issues.Markdownlint/MarkdownlintIssuesAliases/EBFF674A
+[MarkdownlintCliLogFileFormat]: ../../../api/Cake.Issues.Markdownlint/MarkdownlintIssuesAliases/B518F49E
+[IssuePriority.Warning]: ../../../api/Cake.Issues/IssuePriority/7A0CE07F
```
Comment (x-jokay) [inline docs/features.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/requirements.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+Order: 10
+Title: Requirements
+Description: Requirements for the Cake.Issues.Markdownlint addin.
+---
+The requirements for using the [Cake.Issues.Markdownlint addin] are listed in the [release notes] for any specific version.
+
+[Cake.Issues.Markdownlint addin]: https://www.nuget.org/packages/Cake.Issues.Markdownlint
+[release notes]: release-notes
```
Comment (x-jokay) [inline docs/requirements.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/features.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+---
+The [Cake.Issues.Markdownlint addin] provides the following features.
+
+# Basic features
+
+* Reads warnings from [Markdownlint] logfiles.
+* Provides URLs for all issues.
+* Support for custom URL resolving using the [MarkdownlintAddRuleUrlResolver] alias.
+
+# Supported log file formats
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+* [MarkdownlintLogFileFormat] alias for reading issues from [Markdownlint] output generated with `options.resultVersion` set to 1.
+* [MarkdownlintCliLogFileFormat] alias for reading issues from [markdownlint-cli] log files.
+
+  :::{.alert .alert-info}
+  [markdownlint-cli] can be run with the [Cake.Markdownlint] addin.
+  :::
+
+# Supported comment formats
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProviderName`             |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectName`              |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectFileRelativePath`  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.AffectedFileRelativePath` |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Line`                     |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Message`                  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Priority`                 | Always [IssuePriority.Warning] |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.PriorityName`             | Always [IssuePriority.Warning] |
```
Comment (x-jokay) [inline docs/features.md]: `Always [IssuePriority.Warning]` seems not correct for `IIssue.PriorityName`.

File: docs/examples.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    RunMarkdownlintNodeJs(settings);
+});
+```
+
+Finally you can define a task where you call the core addin with the desired issue provider.
+
+```csharp
+Task("Analyze-Log")
+.IsDependentOn("Lint-Documentation")
+.Does(() =>
```
Comment (x-jokay) [inline docs/examples.md]: Fix indention for `.IsDependentOn` and `.Does`.

**Comment by pascalberger** [comment]
@x-jokay Ready for another review.

Top level headings are  required to have the entry appear in site TOC (See https://github.com/cake-contrib/Cake.Issues.Website/pull/41)

**Pull request merged**